year-pages/2019-conferences-list.md

```diff
@@ -5,6 +5,7 @@
 | [ReactFoo](https://reactfoo.in/2019/) | 1-2 Mar | NIMHANS Convention Centre, Bengaluru | A conference on React, alternatives to React, React Native and front-end engineering | No |
 | [SACON](https://www.sacon.io/) | 15-16 Feb | Bengaluru |  India’s 1st Security Architecture Conference | No |
 | [Processing Community Day](https://processingindia.org/bangalore.html) | 2 Feb| Bengaluru | Processing Community Day (PCD) is a day to celebrate and explore art, code, and diversity around the world, conducted by Processing community | No |
+| [PGConf India](https://pgconf.in/conferences/pgconfin2019) | 13-15 Feb | Bengaluru | Conference for Postgresql users and devs | No |
```

Comment:
@indradhanush This will need to go above Processing Community Day because it's after PCD (this repo follows reverse chronological order). :)

Reply:
Sorry bad miss on my part. Updated now. :)

Reply:
Np, thanks for adding the conference!